content/en/docs/setup/additional-setup/requirements/index.md

```diff
@@ -13,28 +13,6 @@ keywords: [kubernetes,sidecar,sidecar-injection]
 To be a part of an Istio service mesh, pods and services in a Kubernetes
 cluster must satisfy the following requirements:
 
-- **Named service ports**: Service ports must be named. The port name key/value
```

Comment:
Instead of just removing this section, maybe it would be good to leave a one liner with a reference to the op guide for more information. For example, something like this:
```
**Named service ports**: For some protocols, service ports must be named to enable certain Istio features. Refer to [Protocol Selection](/docs/ops/traffic-management/protocol-selection) for details.
```

Reply:
I disagree. This page is for *requirements* for using Istio. The whole point of this feature was to remove one more item from this list.

Defining protocols is no longer a requirement for running Istio - it is merely a requirement to get some extra functionality in *rare* cases (mysql, redis, etc are not even out of Alpha - I would guess 99% of traffic is http or tcp which works out of the box now).

Just like we don't say `specify `--set grafana.enabled=true` to get dashboards, it is not appropriate to specify this here as well.

What do you think?

Reply:
Fair enough, although I still wonder if there should be a link from somewhere else. E.g., FAQ entry: "Do service ports need to be named?", with answer, only for certain protocols that are not automatically recognized, See <link>.